src/components/browser/browser.js

```diff
@@ -21,20 +21,14 @@ class Browser extends React.PureComponent {
           </div>
         </div>
         <div className="block-mm none">
-          <div className="h30 bg-gray-faint color-gray flex-parent flex-parent--space-between-main px6 flex-parent--center-cross">
-            <div
-              style={{ width: '70px' }}
-              className="flex-parent-inline flex-parent--space-between-main flex-parent--center-cross mr18"
-            >
+          <div className="h30 bg-gray-faint color-gray flex flex--space-between-main px6 flex--center-cross">
```

Comment:
Curious what you think about using bg-gray-lighter here. It's a new shade, that is in between faint and light. I think it adds a little more contrast.

```suggestion
          <div className="h30 bg-gray-lighter color-gray flex flex--space-between-main px6 flex--center-cross">
```